README.md

```diff
@@ -42,20 +34,20 @@ We highly recommend using Require.js with Pinny. To use Require, you have to ref
 
 {
     'paths': {
-    	'plugin': 'bower_components/plugin/dist/plugin.min',
-        'pinny': 'bower_components/pinny/dist/pinny.min',
-        'modal-center': 'bower_components/pinny/dist/effect/modal-center',
-        'sheet-bottom': 'bower_components/pinny/dist/effect/sheet-bottom',
-        'sheet-left': 'bower_components/pinny/dist/effect/sheet-left',
-        'sheet-right': 'bower_components/pinny/dist/effect/sheet-right',
-        'sheet-top': 'bower_components/pinny/dist/effect/sheet-top',
-        'shade': 'bower_components/shade/dist/shade.min',
-        'lockup': 'bower_components/lockup/dist/lockup.min',
-        'deckard': 'bower_components/deckard/dist/deckard.min',
-        'bouncefix': 'bower_components/bouncefix.js/dist/bouncefix.min'
-        'event-polyfil': 'bower_components/pinny/src/js/utils/event-polyfil',
-        'velocity': 'bower_components/mobify-velocity/velocity.min',
-        'isChildOf': 'bower_components/selector-utils/src/selector/isChildOf',
+    	'plugin': 'node_modules/pinny/node_modules/plugin/dist/plugin.min',
+        'pinny': 'node_modules/pinny/dist/pinny.min',
+        'modal-center': 'node_modules/pinny/effect/modal-center',
+        'sheet-bottom': 'node_modules/pinny/effect/sheet-bottom',
+        'sheet-left': 'node_modules/pinny/effect/sheet-left',
+        'sheet-right': 'node_modules/pinny/effect/sheet-right',
+        'sheet-top': 'node_modules/pinny/effect/sheet-top',
+        'shade': 'node_modules/pinny/node_modules/shade/dist/shade.min',
+        'lockup': 'node_modules/pinny/node_modules/lockup/dist/lockup.min',
+        'deckard': 'node_modules/pinny/node_modules/deckard/dist/deckard.min',
+        'bouncefix': 'node_modules/pinny/node_modules/bouncefix.js/dist/bouncefix.min'
```

Comment:
I just mean that eventually we'll probably want `deckard` to be installed via it's own NPM package. Then we'd be able to specify it's version in package.json. In this file we'd then alias it from `node_modules/deckard/dist/deckard.min` rather than from within the pinny node module.